src/main/java/com/documaster/rms/noark/ws/client/HttpClient.java

```diff
@@ -14,6 +14,8 @@
 	private String authToken;
 	private boolean ignoreCertificateErrors;
 
+	private RmsClient.ErrorResponseType errorResponseType;
```

Comment:
Using RmsClient here makes the superclass depend on it's own subclass. This just feels a bit... weird. 
Dependencies should normally go the other way.

Reply:
I agree. At first glance I am even surprised this did not issue any warnings in my IDE. This is probably a left-over of my earlier attempts.

Fixed. I simply removed the RmsClient prefixes.